ci-operator/step-registry/baremetal/lab/sno-bip/baremetal-lab-sno-bip-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# Trap to kill children processes
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM ERR
+# Save exit code for must-gather to generate junit
+trap 'echo "$?" > "${SHARED_DIR}/install-status.txt"' TERM ERR
+
+# shellcheck disable=SC2154
+installation_disk=$(echo -n "$architecture" | sed 's/arm64/\/dev\/nvme0n1/;s/amd64/\/dev\/sda/')
```

Comment:
The shellcheck can be resolved by adding the default value.
```suggestion
installation_disk=$(echo -n "${architecture:-amd64}" | sed 's/arm64/\/dev\/nvme0n1/;s/amd64/\/dev\/sda/')
```